recite/admin.py

```diff
@@ -112,11 +127,7 @@ def get_file_audio_path(self, file_paths, ayah):
         """Search in uploaded files for corresponding ayah"""
 
         file_ayah_name_expected = str(ayah) + '.mp3'
-        file_ayah = None
-        for file_path in file_paths:
-            if file_ayah_name_expected == os.path.basename(file_path):
-                file_ayah = file_path
-                break
+        file_ayah = file_paths.get(file_ayah_name_expected)
```

Comment:
All of this logic breaks once there is one missing file.
We need to find **all** missing files and output it to the user. Either do it in this PR, or create an issue to address it later.